client/lib/interpolate-components/Makefile

```diff
@@ -1,7 +1,23 @@
 REPORTER ?= spec
-MOCHA ?= ../../../node_modules/.bin/mocha
+
+ifneq ("$(wildcard node_modules/.bin/)","")
+NODE_BIN ?= ./node_modules/.bin
+else
+NODE_BIN ?= ../../../node_modules/.bin
+endif
+
+BABEL ?= $(NODE_BIN)/babel
+MOCHA ?= $(NODE_BIN)/mocha
 
 test:
 	@NODE_ENV=test NODE_PATH=test:../../../client $(MOCHA) --compilers jsx:babel/register --reporter $(REPORTER)
 
+publish:
+	@npm install
+	@make test
+	$(BABEL) -sd lib/ src/
+	@npm publish
+	rm -rf lib
+	rm -rf node_modules
```

Comment:
Does this mean we're going to keep `interpolate-components`'s dependencies installed and versioned in the main repository? Why not keeping them only in this lib's directory, would it cause any problem?
We would probably need a more complicated Makefile which installs dependencies for all libs but certainly this is feasible don't you think?